src/test/pages/customersPage.page.ts

```diff
@@ -35,24 +39,24 @@ export class CustomerPage extends MembersPage {
     }
 
     public async addInvalidEmail() {
-
+      await this.emailInput.fill("  ")
     }
 
     public async checkSaveButtonAvailability() {
         await new Promise(resolve => setTimeout(resolve, 5000));
```

Comment:
what is the purpose of promises? Isn't working default time out set in config file?